etc/docker/travis/aliases-py27.conf

```diff
@@ -2,6 +2,8 @@
 WSGIScriptAlias /accounts                /opt/rucio/lib/rucio/web/rest/account.py
 WSGIScriptAlias /accountlimits           /opt/rucio/lib/rucio/web/rest/account_limit.py
 WSGIScriptAlias /auth                    /opt/rucio/lib/rucio/web/rest/authentication.py
+WSGIScriptAlias /authentication          /opt/rucio/lib/rucio/web/rest/authentication.py
+WSGIScriptAlias /authorisation           /opt/rucio/lib/rucio/web/rest/authorisation.py
```

Comment:
The convention is to have plural for the resource names:  `/authentications` `/authorisations` ?

Reply:
About `/authorisation`, not sure it's the right terminology as it corresponds to get signed url operation. I'm wondering is if it should not go under a specific rse resource operation like `GET /rses/{rse}/signed_rules` 

Unless it will be extended to more use case ?

Reply:
For new endpoints we can point against `webpy/v1/authentication.py` directly, I think with little need to duplicate them as it was done to be backward compatible.

Reply:
I don't think we should point them against the version directly. This is a potential pitfall for people which setup their httpd per hand (and don't use docker). This should still be the standard file which is a symbolic link actually.

Reply:
- Plural doesn't really make sense with this? (like with /redirect or /meta...)

- The signature operation is independent from the RSE, I don't see how/why it should go there? It doesn't have anything to do with rules etc either. Authorisation (--> what am I allowed to do --> issuing bearer tokens and signed urls) is a completely separate functionality.

- Endpoints -> Yeah, I agree.

Reply:
yes, a mixed file with the two convention is bad. For me the main issue is backward compatibility. `aliases-py27.conf` with all 'v1' paths will work and be consistent for example.

Reply:
* Convention: I would also keep singular here and not use plural. It makes sense for entities where there is multiple of them, dids, rules, etc. but for others such as ping, redirect it doesn't make so much sense and for some it just sounds strange (meta).
* path: Even if all are v1 it is still not backward compatible. It might be that you install a new rucio version (new server, new client) and it stops working, as the client uses new rest formats the server does not know. I think the current convention works well here.

Reply:
@mlassnig: About 'The signature operation is independent from the RSE, I don't see how/why it should go there? It doesn't have anything to do with rules etc either. Authorisation (--> what am I allowed to do --> issuing bearer tokens and signed urls) is a completely separate functionality.':
-> It depends how you see it, I guess. An url is valid only for a RSE and can be considered as part of the RSE services like in macaroon/s3 actually. The signed url service can be seen also seen as an independent service. Both views are valid imo.

@bari12: About path, this is getting more complicated if we look at the complete picture and should be discussed when v2 will be introduced wrt compatibility with clients (version header variable, version flag, etc). I would keep it for later...

Reply:
Yes true, to be backward compatible we probably need to serve both endpoints anyway. Or doe something smart based on the client version etc. anyway, to be discussed when relevant 😄

Reply:
@vingar We can add it to the RSE interface, I have to think about the best way. But in any case, we need the separate service endpoint for OAuth2 workflows (e.g., FTS requesting a signed URL for source/destination PFNs).

@bari12 I'll leave the paths now as they are. I'll let you change it ;-)

Reply:
just think about´/authenticate´ and ´/authorise´ are consistent with `/redirect`.

Reply:
openstack calls it /auth for authentication and /credentials for authorisation. that could also work.

Reply:
+1 for `/credentials` or `authenticate`

`auth` is already there;,`authorise` ?

Reply:
i like /credentials. @bari12 opinion?